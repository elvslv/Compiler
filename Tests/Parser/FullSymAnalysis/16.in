var i, j: integer;
function func(i : integer): real; forward;
begin
	if func(10) then
		i := j;
end.