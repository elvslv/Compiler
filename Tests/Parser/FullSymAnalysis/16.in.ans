var I : INTEGER;
var J : INTEGER;
function FUNC(var I : INTEGER;): REAL;

17 4 Condition must be integer
