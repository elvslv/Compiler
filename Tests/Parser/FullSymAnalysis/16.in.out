var I : INTEGER;
var J : INTEGER;

17 4 Condition must be integer